runtime/lua/vim/_editor.lua

```diff
@@ -914,6 +914,13 @@ function vim._init_defaults()
   vim._init_default_autocmds()
 end
 
+local version = require('vim.version')
+vim.version = setmetatable({ cmp = version.cmp, parse = version.parse }, {
+  __call = function()
+    return vim.fn.api_info().version
+  end,
+})
+
```

Comment:
I think this can be moved to `_init_packages.lua`, around line 57

Reply:
If you do it in verision.lua then you need to require in somewhere. By doing it in _init_packages, version.lua won't be loaded until vim.version is accessed, like all the other modules.